src/core/ee/emotiondisasm.cpp

```diff
@@ -1491,6 +1527,22 @@ string EmotionDisasm::disasm_viaddi(uint32_t instruction)
     return output.str();
 }
 
+string EmotionDisasm::disasm_vcallms(uint32_t instruction)
+{
+    stringstream output;
+    uint32_t imm = (instruction >> 6) & 0x7FFF;
+    imm *= 8;
+    output << "vcallms addr 0x" << imm;
```

Comment:
Add `setfill('0') << setw(8) << hex` before "imm" for it to display properly as a hexadecimal address.